packages/fluentui/docs/src/examples/components/Card/Variations/CardExampleFluid.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import { Button, Flex, Image, Text, Avatar, Card, Layout } from '@fluentui/react-northstar';
+import { StarIcon, DownloadIcon, MoreIcon } from '@fluentui/react-icons-northstar';
+
+const CardExample = () => (
+  <Card fluid>
+    <Card.Header>
+      <Flex gap="gap.small">
+        <Avatar image="public/images/avatar/small/matt.jpg" label="Copy bandwidth" name="Evie yundt" status="unknown" />
+        <Flex column>
+          <Text content="Fluid card" weight="bold" />
+          <Text content="Secondary line" size="small" />
+        </Flex>
+      </Flex>
+    </Card.Header>
+    <Card.Body>
+      <Flex column gap="gap.small">
+        <Image src="public/images/wireframe/square-image.png" fluid />
+        <Text content="Content text" />
+      </Flex>
+    </Card.Body>
+    <Card.Footer>
+      <Flex space="between">
+        <Button content="Action" />
+        <Flex>
+          <Button icon={<StarIcon />} iconOnly text title="Favourite" />
+          <Button icon={<DownloadIcon />} iconOnly text title="Download" />
+          <Button icon={<MoreIcon />} iconOnly text title="More" />
+        </Flex>
+      </Flex>
+    </Card.Footer>
+  </Card>
+);
+
+const ImageExampleFluent = () => (
+  <div>
+    <Layout styles={{ maxWidth: '200px' }} debug renderMainArea={() => <CardExample />} />
+    <Layout styles={{ maxWidth: '400px' }} debug renderMainArea={() => <CardExample />} />
+    <Layout styles={{ maxWidth: '600px' }} debug renderMainArea={() => <CardExample />} />
+  </div>
+);
+
+export default ImageExampleFluent;
```

Comment:
```tsx
import { Button, Flex, Image, Text, Avatar, Card, Layout } from '@fluentui/react-northstar'
import { StarIcon, DownloadIcon, MoreIcon } from '@fluentui/react-icons-northstar'
import React from 'react'

const CardExampleFluid = () => (
  <Card fluid>
    <Card.Header>
      <Flex gap="gap.small">
        <Avatar
          image="public/images/avatar/small/matt.jpg"
          label="Copy bandwidth"
          name="Evie yundt"
          status="unknown"
        />
        <Flex column>
          <Text content="Fluid card" weight="bold" />
          <Text content="Secondary line" size="small" />
        </Flex>
      </Flex>
    </Card.Header>
    <Card.Footer>
      <Flex space="between">
        <Button content="Action" />
        <Flex>
          <Button icon={<StarIcon />} iconOnly text title="Favourite" />
          <Button icon={<DownloadIcon />} iconOnly text title="Download" />
          <Button icon={<MoreIcon />} iconOnly text title="More" />
        </Flex>
      </Flex>
    </Card.Footer>
  </Card>
)

export default CardExampleFluid
```

I suggest to simplify this example:

![image](https://user-images.githubusercontent.com/14183168/78791578-5d047680-79b0-11ea-9516-e622391204f9.png)